eph/scheduler.go

```diff
@@ -83,6 +85,9 @@ func (s *scheduler) Run(ctx context.Context) {
 		case <-ctx.Done():
 			s.dlog(ctx, "shutting down scan")
 			return
+		case <-s.close:
```

Comment:
In line 188, we actually call the s.done() function, which cancels the context we used to spin up this entire thing. So I think this s.close is not needed.